### PR TITLE
update with new colors

### LINKED ISSUE #81: Docs Theme Adjustments
- [ ] White background
- [ ] Sidebar color
- [ ] Etc… from design

NOTE: Dark mode is NOT changing

Image

### PR DESCRIPTION
**Pull request by shannonwells**
Problem
=======
Theme adjustments for MD book to reflect current mocks

Closes #81 

Steps to Verify:
----------------
```shell
git fetch && git checkout feat/update-colors
npm install
npm run serve
```
1. Verify that the overall look is like the Figma mockups in the linked issue.
2. Make sure the SVGs have a matching background
3. This is less certain, but I also changed appearances of some links to try to match the mockups, which don't have links.  This is likely going to change so based on @wilwade's input it's less important given that it's in a branch.

Screenshots (optional):
-----------------------
colors

### CODE REVIEW

File: theme/css/chrome.css [hunk 1]
```diff
@@ -138,8 +142,6 @@ a > .hljs {
 .menu-bar:visited,
 .nav-chapters,
 .nav-chapters:visited,
-.mobile-nav-chapters,
```
Comment (shannonwells) [inline theme/css/chrome.css:141]: These two being here was making it impossible to see button links and some links within text.  The style was/is applied regardless of whether it's actually mobile size.  I didn't correct that part, just made it visible.

File: pages/images/ParachainModel-Dark.svg
Comment (shannonwells) [inline pages/images/ParachainModel-Dark.svg:1]: There was a lot of the "cream" color left over from a previous update. This was converted to white.

File: pages/images/ParachainModel.svg
Comment (shannonwells) [inline pages/images/ParachainModel.svg:1]: Same here but it's more obvious from the picture.

**Review by wilwade — APPROVED** [review]
These changes look good an no reason to block them. A few questions however:

1. The Frequency Logo is hidden, but I assume that is due to the static header that will change that.
2. Do we still want the left/right nav buttons? If so, I think we might want to to switch to the style guide's primary (teal) buttons, but the hover on those doesn't work on white currently. @claireolmstead might have some thoughts?
3. The Style Light/Dark dropdown in Light mode is all dark currently, although we don't know if that is staying around either.
4. We likely need a new color for the active mode on the sidebar in light mode. @claireolmstead ideas?

**Pull request merged**